cpp/src/aztec/honk/pcs/gemini/gemini.hpp

```diff
@@ -90,6 +90,12 @@ template <typename Params> struct Proof {
  */
 template <typename Params> using OutputClaim = std::vector<OpeningClaim<Params>>;
 
+/**
+ * @brief Vector of opening pairs {r, v = p(r)}
+ * @tparam Params CommitmentScheme parameters
+ */
+template <typename Params> using OutputPair = std::vector<OpeningPair<Params>>;
+
```

Comment:
I'm not entirely sure this alias is required (as well as `OutputWitnesse`). It might be clearer to use the `vector` directly in `ProverOutput`. The comments for the types could just be part of the member variable definitions in `ProverOutput`.

Reply:
Agreed, I almost did this but then didn't for some reason. I've now removed all `Output*` types